python/wtte/transforms.py

```diff
@@ -2,15 +2,19 @@
 from __future__ import division
 from __future__ import print_function
 
+import sys
```

Comment:
I didn't know that it's a bad practice, could you give a link for that (I couldn't google it). But when I searched for checking python version like [SO question](https://stackoverflow.com/q/1093322/4542359) there are a lot of answers using `sys`.
Nevertheless it's a good change without depending on version. 
Is the `try` and `except` better then importing `sys`?